eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -57,14 +77,12 @@
     <_ApiCompatSemaphoreFile>$(MSBuildThisFileName).semaphore</_ApiCompatSemaphoreFile>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(RunWpfApiCompat)' == 'true'">
+  <PropertyGroup Condition="'$(RunNetFrameworkApiCompat)' == 'true' or '$(RunRefApiCompat)'=='true'">
     <_apiCompatTargetSuffix>$(TargetGroup)</_apiCompatTargetSuffix>
     <_apiCompatTargetSuffix Condition="'$(_apiCompatTargetSuffix)' == ''">$(TargetFramework)</_apiCompatTargetSuffix>
 
-    <ApiCompatBaseline Condition="!Exists('$(ApiCompatBaseline)')">$(MSBuildProjectDirectory)\ApiCompatBaseline.$(_apiCompatTargetSuffix).txt</ApiCompatBaseline>
-    <ApiCompatBaseline Condition="!Exists('$(ApiCompatBaseline)')">$(MSBuildProjectDirectory)\ApiCompatBaseline.txt</ApiCompatBaseline>
-
-    <TargetsTriggeredByCompilation Condition="'$(RunWpfApiCompat)' == 'true'">$(TargetsTriggeredByCompilation);WpfValidateApiCompatForSrc</TargetsTriggeredByCompilation>
+    <TargetsTriggeredByCompilation Condition="'$(RunNetFrameworkApiCompat)' == 'true'">$(TargetsTriggeredByCompilation);WpfValidateApiCompatForNetFramework</TargetsTriggeredByCompilation>
+    <AfterLinkTargets Condition="'$(RunRefApiCompat)' == 'true'">$(AfterLibTargets);WpfValidateApiCompatForRef</AfterLinkTargets>
```

Comment:
Add comment on why we need AfterLinkTargets here instead of TargetsTiggeredByCompilation.

We probably should also condition these on ManagedCxx so that we use AfterLinkTargets when dealing with a lib project that is C++\CLI and TargetsTriggeredByCompilation when dealing with C#.  This will allow this to continue working if we pivot to ref assemblies generated by GenAPI.